docs/understanding-server-virtualization/understanding-hypervisors.rst

```diff
@@ -1,14 +1,13 @@
 .. _understanding_hypervisors:
 
-2.2
 
 =========================
 Understanding hypervisors
 =========================
 
-A hypervisor is an OS that runs on a physical server and pools the 
-available physical resources into the virtual resources that VMs require. 
-You use a hypervisor to run and manage your VMs. This document refers 
-to VMware ESXi™, which is a type of hypervisor. The word hypervisor can 
+A hypervisor is an OS that runs on a physical server and pools the
+available physical resources into the virtual resources that VMs require.
+You use a hypervisor to run and manage your VMs. This document refers
+to VMware ESXi™, which is a type of hypervisor. The word hypervisor can
 also refer to the physical machine on which ESXi is installed.
-__________________________________
\ No newline at end of file
+__________________________________ 
```

Comment:
What are these lines at the end of each file? Did you add them or was it an automatic VScode thing?

Reply:
No, I did it.  I'll get rid of them though. Thank you for taking a look at the PR Steph. =)